scripts/linux/runE2ETest.sh

```diff
@@ -25,6 +25,9 @@ function clean_up() {
     if [ "$CLEAN_ALL" = '1' ]; then
         echo 'Prune docker system'
         docker system prune -af --volumes || true
+
+        echo 'Restart docker'
+        systemctl restart docker
```

Comment:
This change is needed for testing and also exists in another PR to master that will be merged soon.